pkgs/by-name/sc/scalar/package.nix

```
@@ -0,0 +1,67 @@
+{
+  lib,
+  stdenv,
+  fetchurl,
+  makeWrapper,
+  makeDesktopItem,
+  copyDesktopItems,
+  electron,
+  dpkg,
+}:
+
+stdenv.mkDerivation rec {
+  pname = "scalar";
+  version = "unstable-2025-05-04";
```

Unstable version are prefixed by the major version
```suggestion
  version = "0-unstable-2025-05-04";
```